Launching error after fresh install

Hi there,

I just installed ungit after fresh installation of Git 2.9.0 and Node.js 4.4.7 on Windows 10: 

```
PS C:\Users\lebrach> npm install -g ungit
npm WARN engine forever-monitor@1.1.0: wanted: {"node":"0.8.x"} (current: {"node":"4.4.7","npm":"2.15.8"})
npm WARN deprecated minimatch@0.0.5: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN engine hawk@0.10.2: wanted: {"node":"0.8.x"} (current: {"node":"4.4.7","npm":"2.15.8"})
npm WARN engine boom@0.3.8: wanted: {"node":"0.8.x"} (current: {"node":"4.4.7","npm":"2.15.8"})
npm WARN engine cryptiles@0.1.3: wanted: {"node":"0.8.x"} (current: {"node":"4.4.7","npm":"2.15.8"})
npm WARN engine hoek@0.7.6: wanted: {"node":"0.8.x"} (current: {"node":"4.4.7","npm":"2.15.8"})
npm WARN engine sntp@0.1.4: wanted: {"node":"0.8.x"} (current: {"node":"4.4.7","npm":"2.15.8"})
C:\Users\lebrach\AppData\Roaming\npm\0ungit-credentials-helper -> C:\Users\lebrach\AppData\Roaming\npm\node_modules\
ungit\bin\credentials-helper
C:\Users\lebrach\AppData\Roaming\npm\ungit -> C:\Users\lebrach\AppData\Roaming\npm\node_modules\ungit\bin\ungit
ungit@0.10.2 C:\Users\lebrach\AppData\Roaming\npm\node_modules\ungit
├── os-homedir@1.0.1
├── open@0.0.5
├── crossroads@0.12.2
├── knockout@3.4.0
├── semver@5.1.1
├── signals@1.0.0
├── blueimp-md5@2.3.0
├── cookie-parser@1.4.3 (cookie-signature@1.0.6, cookie@0.3.1)
├── passport-local@1.0.0 (passport-strategy@1.0.0)
├── passport@0.3.2 (pause@0.0.1, passport-strategy@1.0.0)
├── keen.io@0.1.3 (underscore@1.5.2)
├── temp@0.8.3 (os-tmpdir@1.0.1, rimraf@2.2.8)
├── color@0.11.3 (color-convert@1.3.1, clone@1.0.2, color-string@0.3.0)
├── rc@1.1.6 (ini@1.3.4, deep-extend@0.4.1, strip-json-comments@1.0.4, minimist@1.2.0)
├── express-session@1.13.0 (cookie-signature@1.0.6, utils-merge@1.0.0, on-headers@1.0.1, parseurl@1.3.1, cookie@0.2.3, d
epd@1.1.0, crc@3.4.0, uid-safe@2.0.0, debug@2.2.0)
├── mkdirp@0.5.1 (minimist@0.0.8)
├── bluebird@3.3.5
├── serve-static@1.10.3 (escape-html@1.0.3, parseurl@1.3.1, send@0.13.2)
├── diff2html@1.2.0 (diff@2.2.3)
├── body-parser@1.15.2 (content-type@1.0.2, bytes@2.4.0, depd@1.1.0, qs@6.2.0, on-finished@2.3.0, raw-body@2.1.7, http-e
rrors@1.5.0, debug@2.2.0, iconv-lite@0.4.13, type-is@1.6.13)
├── raven@0.11.0 (lsmod@1.0.0, stack-trace@0.0.7, cookie@0.1.0, node-uuid@1.4.7)
├── winston@2.2.0 (cycle@1.0.3, stack-trace@0.0.9, eyes@0.1.8, isstream@0.1.2, async@1.0.0, pkginfo@0.3.1, colors@1.0.3)

├── express@4.13.4 (escape-html@1.0.3, array-flatten@1.1.1, cookie-signature@1.0.6, utils-merge@1.0.0, methods@1.1.2, co
ntent-type@1.0.2, vary@1.0.1, parseurl@1.3.1, cookie@0.1.5, range-parser@1.0.3, fresh@0.3.0, etag@1.7.0, path-to-regexp@
0.1.7, content-disposition@0.5.1, merge-descriptors@1.0.1, depd@1.1.0, qs@4.0.0, on-finished@2.3.0, finalhandler@0.4.1,
debug@2.2.0, proxy-addr@1.0.10, send@0.13.1, type-is@1.6.13, accepts@1.2.13)
├── rimraf@2.5.2 (glob@7.0.5)
├── getmac@1.2.1 (extract-opts@3.3.1)
├── superagent@0.21.0 (extend@1.2.1, cookiejar@2.0.1, methods@1.0.1, mime@1.2.11, component-emitter@1.1.2, reduce-compon
ent@1.0.1, qs@1.2.0, debug@2.2.0, readable-stream@1.0.27-1, form-data@0.1.3, formidable@1.0.14)
├── async@2.0.0-rc.6
├── socket.io@1.4.8 (has-binary@0.1.7, debug@2.2.0, socket.io-parser@2.2.6, socket.io-adapter@0.4.0, engine.io@1.6.11, s
ocket.io-client@1.4.8)
├── yargs@4.7.1 (decamelize@1.2.0, camelcase@3.0.0, y18n@3.2.1, set-blocking@1.0.0, require-main-filename@1.0.1, window-
size@0.2.0, yargs-parser@2.4.0, lodash.assign@4.0.9, cliui@3.2.0, string-width@1.0.1, os-locale@1.4.0, pkg-conf@1.1.3, r
ead-pkg-up@1.0.1)
├── octicons@3.5.0
├── hasher@1.2.0
├── npm-registry-client@7.1.2 (chownr@1.0.1, graceful-fs@4.1.4, slide@1.1.6, retry@0.8.0, once@1.3.3, npm-package-arg@4.
2.0, concat-stream@1.5.1, normalize-package-data@2.3.5, npmlog@3.1.2, request@2.72.0)
├── snapsvg@0.4.0 (eve@0.4.2)
├── forever-monitor@1.1.0 (watch@0.5.1, pkginfo@0.4.0, minimatch@0.0.5, utile@0.1.7, ps-tree@0.0.3, broadway@0.2.10)
├── moment@2.13.0
└── lodash@4.12.0
PS C:\Users\lebrach>
```
And got following error when launching ungit:

```
PS C:\Users\lebrach> ungit
module.js:327
    throw err;
    ^

Error: Cannot find module 'npm'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\Users\lebrach\AppData\Roaming\npm\node_modules\ungit\src\sysinfo.js:10:11)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

Don't know what to do do now...

Any help would be appreciated :-)

-- 
Lebrac'h

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,9 @@ This project adheres to [Semantic Versioning](http://semver.org/) and
 
 ## [Unreleased](https://github.com/FredrikNoren/ungit/compare/v0.10.2...master)
 
+### Fixed
+- Missing npm as a normal dependency [#766] (https://github.com/FredrikNoren/ungit/issues/766)
+
 ## [0.10.2](https://github.com/FredrikNoren/ungit/compare/v0.10.1...v0.10.2)
 
 ### Added

--- a/package.json
+++ b/package.json
@@ -35,6 +35,7 @@
     "lodash": "~4.12.0",
     "mkdirp": "~0.5.1",
     "moment": "~2.13.0",
+    "npm": "~3.9.1",
     "npm-registry-client": "~7.1.0",
     "octicons": "~3.5.0",
     "open": "~0.0.5",
@@ -77,7 +78,6 @@
     "grunt-release": "~0.13.1",
     "istanbul": "~0.4.3",
     "mocha": "~2.4.5",
-    "npm": "~3.9.1",
     "phantomjs-prebuilt": "^2.1.7",
     "supertest": "~0.15.0"
   },

